### PR TITLE
[Double Brace] Fix standard approach in the samples solution

### PR DESCRIPTION
**Pull request by FedericoBernal**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/51.teams-messaging-extensions-action/src/main/java/com/microsoft/bot/sample/teamsaction/TeamsMessagingExtensionsActionBot.java [hunk 1]
```diff
@@ -78,7 +78,6 @@
         card.setText(action.getMessagePayload().getBody().getContent());
 
         CardImage cardImage = new CardImage();
```
Comment (Batta32) [inline samples/51.teams-messaging-extensions-action/src/main/java/com/microsoft/bot/sample/teamsaction/TeamsMessagingExtensionsActionBot.java:80]: This should be inside the `if` sentence too

File: samples/52.teams-messaging-extensions-search-auth-config/src/main/java/com/microsoft/bot/sample/teamssearchauth/TeamsMessagingExtensionsSearchAuthConfigBot.java [hunk 2]
```diff
@@ -190,24 +189,28 @@ public TeamsMessagingExtensionsSearchAuthConfigBot(
...
+                            suggestedAction.setAction(cardAction);
 
-                    extensionResponse.setComposeExtension(result);
+                            MessagingExtensionResult result = new MessagingExtensionResult();
+                            result.setType("auth");
+                            result.setSuggestedActions(suggestedAction);
 
-                    return tokenProvider.getOAuthSignInLink(turnContext, connectionName)
-                        .thenApply(link -> extensionResponse);
+
```
Comment (Batta32) [inline samples/52.teams-messaging-extensions-search-auth-config/src/main/java/com/microsoft/bot/sample/teamssearchauth/TeamsMessagingExtensionsSearchAuthConfigBot.java:209]: Remove empty line

File: samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Application.java [hunk 3]
```diff
@@ -19,8 +19,9 @@
 import com.microsoft.bot.integration.SkillHttpClient;
 import com.microsoft.bot.integration.spring.BotController;
 import com.microsoft.bot.integration.spring.BotDependencyConfiguration;
+import com.microsoft.bot.sample.dialogrootbot.Bots.RootBot;
 import com.microsoft.bot.sample.dialogrootbot.authentication.AllowedSkillsClaimsValidator;
-import com.microsoft.bot.sample.dialogrootbot.bots.RootBot;
+
```
Comment (Batta32) [inline samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Application.java:24]: Remove empty line

File: samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Application.java [hunk 4]
```diff
@@ -19,8 +19,9 @@
 import com.microsoft.bot.integration.SkillHttpClient;
 import com.microsoft.bot.integration.spring.BotController;
 import com.microsoft.bot.integration.spring.BotDependencyConfiguration;
+import com.microsoft.bot.sample.dialogrootbot.Bots.RootBot;
```
Comment (Batta32) [inline samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Application.java:22]: I'm not sure to add these changes in the PR, we are not fixing the DBI, we need to rollback this

File: samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Bots/RootBot.java [hunk 5]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MT License.
 
-package com.microsoft.bot.sample.dialogrootbot.bots;
+package com.microsoft.bot.sample.dialogrootbot.Bots;
```
Comment (Batta32) [inline samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Bots/RootBot.java:4]: I'm not sure to add these changes in the PR, we are not fixing the DBI, we need to rollback this

**Review by Batta32 — CHANGES_REQUESTED** [review]
nit changes.

Also, as mentioned in Slack, we are missing DBIs in the following samples:
- 03.welcome-user (WelcomeUserBot class)
- 08.suggested-actions (SuggestedActionsBot class)
- 81.skills-skilldialog (RootBot class)

After applying these changes, re-validate the samples!

File: samples/51.teams-messaging-extensions-action/src/main/java/com/microsoft/bot/sample/teamsaction/TeamsMessagingExtensionsActionBot.java (hunk 1 above)
Comment (FedericoBernal) [reply samples/51.teams-messaging-extensions-action/src/main/java/com/microsoft/bot/sample/teamsaction/TeamsMessagingExtensionsActionBot.java:80]: Ack, but this is used in both if sentences so I will have to declare it two times

**Review by Batta32 — APPROVED** [review]

**Pull request merged**